flight/Modules/Airspeed/baro_airspeed_analog.c

```diff
@@ -124,6 +124,10 @@ void baro_airspeedGetAnalog(BaroAirspeedData *baroAirspeedData, uint32_t *lastSy
 		//not. This is something that will have to change on the ADC side of things.
 		baroAirspeedData->SensorValue=PIOS_MPXV5004_Measure(airspeedADCPin);
 
+	} else if (airspeedSensorType == AIRSPEEDSETTINGS_AIRSPEEDSENSORTYPE_RAWANALOG) {
+		// will return raw ADC reading in mV (typically in range [0, 3300])
```

Comment:
I think this is 0,4095

Reply:
Ahhh, I may have been thinking of GetVolts.